src/yarle.ts

```diff
@@ -32,25 +33,31 @@ export const parseStream = async (options: YarleOptions): Promise<void> => {
 
       return reject();
     };
-  
+
     xml.collect('tag');
     xml.collect('resource');
-    xml.on('startElement: en-export', (enExport: any) => {
-      totalNotes = Array.isArray(enExport.note) ? enExport.note.length : 1;
```

Comment:
Could you please keep totalNotes, just to store how many notes found in the enex file at all, independently from how many of them were skipped during the conversion?

Reply:
But `notenumber` is already the amount of total notes, no? Do we need to know the total amount before we finish processing them? Because otherwise `notenumber` and `totalNotes` will report the same thing, or at least: _it would_. But `enExport.note` is actually undefined. See my comment on the closed pr #71  

>The current implementation of the console.log in the end of the conversion shows a negative number for failed notes if the enex includes more than one note. The reason is that `enExport.note` in this code snippet

> ```ts
>     xml.on('startElement: en-export', (enExport: any) => {
>       totalNotes = Array.isArray(enExport.note) ? enExport.note.length : 1;
>     });
> ```

>Was never an array (it was actually undefined) and therefore `totalNotes` was always 1. In any case we count the notes through noteNumber so this is actually not needed.

But maybe I'm missing something else?

Reply:
Ah, got it. Only the skipped counter was incremented after my change.

```ts
      if (options.skipWebClips && isWebClip(note)) {
          ++skipped;
          ++noteNumber;
          console.log(`Notes skipped: ${skipped}`);
      } else {
        processNode(note);
        ++noteNumber;
        console.log(`Notes processed: ${noteNumber}`);
      }
```

Would this be an okay solution? I would also make sure that the number for succeeded notes would reflect skipped and failed notes and not only failed notes.

Reply:
Argh. Sorry, this won't be good either. You know what: I will code it first and comment later. 😅

Reply:
Please check my current implementation without the following. I will totally change it though, if you want. The reason why `enExport.note` is undefined is because `enExport.note` seems only to be available on `endElement` like so:

```ts
    xml.on('endElement: en-export', (enExport: any) => {
      const totalNotes = Array.isArray(enExport.note) ? enExport.note.length : 1;
    }); note enExport.note
```

Would you like me to change it in this way?